paasta_itests/paasta_metastatus.feature

```diff
@@ -14,7 +14,7 @@ Feature: paasta_metastatus describes the state of the paasta cluster
   Scenario: High memory usage
     Given a working paasta cluster
      When an app with id "memtest" using high memory is launched
-      And a task belonging to the app with id "memtest" is in the task list
+      And a task belonging to the app with id "/memtest" is in the task list
```

Comment:
This is wrong imo; the id is not "/memtest" -- that would be confusing since we just in the previous step said the id was "memtest". If marathon adds the leading /, our step code should also add it.